### DISCUSSION TITLE
Hotkey support

### DISCUSSION THREAD

**Discussion by sxnvmqe** [sorayuki/obs-multi-rtmp#385] [Ideas]
It would be nice to be able to add their own hotkey per target.

**Comment by Yakka-2k** [comment]
I agree, Hotkey support would be a great feature! 
Ideally, I'd love to set the 'Start all" and "Stop all" to individual hotkeys, and in turn, be able to assign those as buttons on my StreamDeck to make for a better workflow.

**Comment by edwinallan** [comment]
yes hotkeys would be awesome!
